scripts/tasks.sh

```diff
@@ -396,8 +396,14 @@ function get_weave_version() {
     if [ -z "$weave_version" ]; then
         if [ -n "$DOCKER_VERSION" ]; then
             weave_version=$(docker image ls | grep weaveworks/weave-npc | awk '{ print $2 }' | head -1)
+            if [ -z "$weave_version" ]; then
+                weave_version=$(docker image ls | grep kurlsh/weave-npc | awk '{ print $2 }' | head -1)
+            fi
```

Comment:
Curious what @emosbaugh thinks about this, could `docker image ls` show both images (I'm thinking in cases of an upgrade)? If so, do we want to default to the kurlsh image as that should be the more 'up-to-date' on?

Reply:
good point. agreed

Reply:
I swapped them, this is now checking for the kurlsh image first.